cmd/platform/user.go

```diff
@@ -37,12 +37,11 @@ var userDeactivateCmd = &cobra.Command{
 }
 
 var userCreateCmd = &cobra.Command{
-	Use:   "create",
-	Short: "Create a user",
-	Long:  "Create a user",
-	Example: `  user create --email user@example.com --username userexample --password Password1
-  user create --firstname Joe --system_admin --email joe@example.com --username joe --password Password1`,
-	RunE: userCreateCmdF,
+	Use:     "create",
+	Short:   "Create a user",
+	Long:    "Create a user",
+	Example: `  user create --email user@example.com --username userexample --password Password1`,
```

Comment:
It was unnecessary. The full set of options are documented in the --h text. The remaining example shows the minimum set of options required for the command to work. Anything else is optional and is marked as such.